src/Nelmio/Alice/Fixtures/Loader.php

```diff
@@ -245,8 +250,8 @@ public function addPopulator(Populator\Methods\MethodInterface $populator)
     /**
      * parses a file at the given filename
      *
-     * @param string filename
-     * @return array data
+     * @param  string $filename
+     * @return array  data
      */
     protected function parseFile($filename)
```

Comment:
Maybe add a mention on which file are supported? If that's not the place for it add a link to where to find this.